internal/k8s/ako_init.go

```diff
@@ -703,39 +703,56 @@ func (c *AviController) FullSyncK8s() error {
 		}
 	}
 
-	svcObjs, err := utils.GetInformers().ServiceInformer.Lister().Services(metav1.NamespaceAll).List(labels.Set(nil).AsSelector())
+	// Get the accepted Namespace for listing the ingress, route, and service objects present in the namespace.
+	nsObjs, err := utils.GetInformers().NSInformer.Lister().List(labels.Set(nil).AsSelector())
 	if err != nil {
-		utils.AviLog.Errorf("Unable to retrieve the services during full sync: %s", err)
+		utils.AviLog.Errorf("Unable to retrieve the namespaces during full sync: %s", err)
 		return err
 	}
-	for _, svcObj := range svcObjs {
-
-		isSvcLb := isServiceLBType(svcObj)
-		var key string
-		svcLabel := utils.ObjKey(svcObj)
-		ns := strings.Split(svcLabel, "/")
-		if isSvcLb && !lib.GetLayer7Only() {
-			/*
-				Key added to Ingestion queue if
-				1. Advance L4 enabled or
-				2. Namespace is valid
-			*/
-			if !utils.IsServiceNSValid(ns[0]) {
-				continue
+
+	acceptedNamespaces := make(map[string]struct{})
+	for _, nsObj := range nsObjs {
```

Comment:
populateNamespaceList() already prepare a list of valid namespace. We can use that.

Reply:
I have added a function to get all valid namespaces present in the filter.